components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -565,7 +568,13 @@ public enum ErrorMessages {
                 "deletion service with response code : %d."),
         ERROR_CODE_ERROR_GETTING_TENANT_DELETION_ACCESS_TOKEN("65106", "Error while generating access " +
                 "token for tenant deletion service.", "Failed to generate authentication token for client " +
-                "credentials");
+                "credentials"),
+        ERROR_CODE_ERROR_CHECKING_SIBLING_ORGANIZATION_BY_NAME("65107", "Error while checking sibling " +
+                "organization by name of parent organization id: %s.", "Failed to check sibling organization " +
+                "by name."),
+        ERROR_CODE_ERROR_CHECKING_CHILD_ORGANIZATION_BY_NAME("65108", "Error while checking child " +
```

Comment:
`Error while checking child organization by name of parent organization id: %s.` -> `Error while checking child organization of the parent organization id: %s by name`

Reply:
Fixed by - https://github.com/wso2-extensions/identity-organization-management-core/pull/65/commits/a0a69184239b61c5b79cad296fbc282a0a0b380c#diff-ae3ba072650382e9c7b02b03bc5cb1471381a2d7a8c1f9e1bc0d3d547dd2652f